app/commands.py

```diff
@@ -115,3 +115,33 @@ def update_notification_international_flag(self):
             print('commit 250000 updates at {}'.format(datetime.utcnow()))
             db.session.commit()
             result_history = db.session.execute(subq_history).fetchall()
+
+    def fix_notification_statuses_not_in_sync(self):
+        """
+        This will be used to correct an issue where Notification._status_enum and NotificationHistory._status_fkey
+        became out of sync. See 979e90a.
+
+        Notification._status_enum is the source of truth so NotificationHistory._status_fkey will be updated with
+        these values.
+        """
+        MAX = 10000
```

Comment:
Why?

The message prints out "Committed x at y" where x is the number of updates and y is the time. MAX is used, like the scripts before, to indicate the MAX number of notifications to process.

Really don't see the issue here.